test/src/pythontests/test_pitchprobabilisticyin_standard.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2006-2018  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Essentia
+#
+# Essentia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+
+import essentia
+import essentia.standard as ess
+import librosa
+import time
+import os
+
+filename = os.path.join(os.path.dirname(__file__) ,'../../audio/recorded', 'long_voice.wav')
```

Comment:
This audio should be added to the ```essentia-audio``` repository (do a pull request). What is the origin of the audio (license)?

Reply:
We can use the existing audios in essentia-audio. How can I use the audios there? How we add the link in this test .py?

Reply:
Create a pull request with new audio files to add to [essentia-audio](https://github.com/MTG/essentia-audio).

Reply:
pull request sent.